scripts/update-readme-content.ts

```diff
@@ -0,0 +1,186 @@
+#!yarn ts-node
```

Comment:
- Retrieves workspace information
- Generates the dependency graph
- Generates the package list
- Updates the README with both